host/xtest/regression_1000.c

```diff
@@ -34,7 +35,9 @@
 #include <ta_miss_test.h>
 #include <ta_sims_keepalive_test.h>
 #include <ta_concurrent.h>
+#include <ta_tpm_log_test.h>
 #include <sdp_basic.h>
+#include <conf.h>
```

Comment:
Why `<conf.h>`?

Reply:
I wanted to add conditional compilation on CFG_CORE_TPM_EVENT_LOG to avoid IBART fails if OP-TEE wasn't built with support for the TPM event log.

Reply:
The `#ifdef CFG_CORE_TPM_EVENT_LOG` is fine, but `#include <conf.h>` is useless due to:
```
host/xtest/Makefile:159:CFLAGS += -include conf.h
```